src/ripple/shamap/tests/FetchPack.test.cpp

```diff
@@ -89,7 +90,7 @@ class FetchPack_test : public beast::unit_test::suite
         for (int d = 0; d < 3; ++d)
             s.add32 (r.nextInt ());
         return std::make_shared <Item> (
-            to256(s.getRIPEMD160()), s.peekData ());
+            s.getSHA512Half(), s.peekData ());
```

Comment:
The only reference to `make_random_item` is `add_random_items`, and all the calls to `add_random_items` are commented out, so you can remove both of these functions.